<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageToIcoPlugin</class>
 <widget class="QWidget" name="ImageToIcoPlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>输入目录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LFileLineEdit" name="inLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="4" column="3">
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>输入图片名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>输出ICO名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>是否完成</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>输出大小</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openOutputButton">
       <property name="text">
        <string>打开输出目录</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>输出目录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LFileLineEdit" name="outLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="3" column="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="format">
      <string>%v/%m</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LFileLineEdit</class>
   <extends>QLineEdit</extends>
   <header>LWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
